go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/manifoldco/promptui v0.9.0
 	github.com/mitchellh/mapstructure v1.5.0
 	github.com/multiformats/go-multiaddr v0.8.0
-	github.com/pokt-network/smt v0.4.0
+	github.com/pokt-network/smt v0.5.0-beta
```

Comment:
On the offchance you don't know about it, Golang has a native way to "replace" dependencies.

Might be useful for your local development if you don't already use it.

https://go.dev/ref/mod#go-mod-file

![Screenshot 2023-05-30 at 12 25 36 PM](https://github.com/pokt-network/pocket/assets/1892194/7cd8c688-738b-4a88-b35e-bc9503041d5f)